### PR TITLE
Metadata for v0.2.0

### PR DESCRIPTION
**Pull request by WilboMo**
**Issue number:**
N/A

**Description of changes:**
Updates CHANGELOG.md, README.md to reflect changes in the release and updates the default version for Bottlerocket ECS updater. 

**Testing done:**
Deployed stack with CloudFormation template and verified new default was applied to parameters as expected. CloudTrail event excerpt displaying the new default image: 

```
"image": "public.ecr.aws/bottlerocket/bottlerocket-ecs-updater:v0.2.0",
```

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# 0.2.0
+
+* Add pagination functionality.
```
Comment (samuelkarp) [inline CHANGELOG.md]: Pagination is the mechanism that we've added support for larger clusters, but not the feature.

```suggestion
* Add support for clusters larger than 50 container instances.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+* Add after-action summary and done message to log output.
+* Add check to determine if Fargate task is already running.
```
Comment (samuelkarp) [inline CHANGELOG.md]:
```suggestion
* Add check to reduce the chance of concurrent updater runs.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+* Add additional unit tests.
```
Comment (samuelkarp) [inline CHANGELOG.md]: This one isn't a feature.

```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,14 @@
...
+* Update `sendCommand` functionality.
```
Comment (samuelkarp) [inline CHANGELOG.md]: What does this mean?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,14 @@
...
+* Update default Bottlerocket ECS updater version.
```
Comment (samuelkarp) [inline CHANGELOG.md]: I don't think this one is a feature either.

```suggestion
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,14 @@
...
+
+Note: In the Bottlerocket ECS updater v0.1.0 release, support for clusters was limited to 50 container instances. In this release, clusters larger than 50 container instances are now supported.
```
Comment (samuelkarp) [inline CHANGELOG.md]: Might be worth including a :tada:?

File: CHANGELOG.md (hunk 4 above)
Comment (WilboMo) [reply CHANGELOG.md]: I may actually remove it since its not really feature related like some of the other callouts you've made.

**Review by samuelkarp — APPROVED** [review]

**Review by gthao313 — APPROVED** [review]

**Pull request merged**